### ISSUE TITLE
think actions broken in cuisine

### ISSUE THREAD

**Issue by despiegk** [Jumpscale/jumpscale_core8#246]
lots of methods are executed over & over again
the actions optimization avoiding to do things double is broken

**Comment by mie00** [comment]
Probably they are executed again because the action decorator has a kwarg `force=True` which forces the actions to be reexecuted when called again.

**Comment by despiegk** [comment]
so I guess we need to put it on false & test if it works fine
today its way too slow
of make sure the required actions are called with force=False

**Comment by mie00** [comment]
Manually testing it actions when `force=False`, it seems to be working fine

However, these methods are the methods that are defined with `force=True`, and for them that seems logical
```
/cuisine/CuisineCore.py:
sudo, run, run_script, tmux_execute_jumpscript, execute_jumpscript, execute_python

/cuisine/CuisineSSH.py:
test_login, test_login_pushkey, keygen, authorize, unauthorize, unauthorizeAll, sshagent_add, sshagent_remove

/cuisine/apps/CuisinePortal.py:
start, stop

/avahi/Avahi.py:
reload, removeService, getServices
```

Also there were missing decorators on some cuisine application builders which should be fixed by #272, but I think they are not directly related to this issue.

So if the problem still exists, please point out how it can be reproduced.
